Sources/BrowserServicesKit/RemoteMessaging/Mappers/JsonRemoteMessageMapper.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  JsonRemoteMessageMapper.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import os.log
+
+private enum AttributesKey: CaseIterable {
+    static let locale             = "locale"
+    static let osApi              = "osApi"
+    static let flavor             = "flavor"
```

Comment:
This isn't really a meaningful name to me, is it standard language in android land?
If so, I think this would be a good point to deviate from them since it's not in apple land

Reply:
Yep, it is. Check out `AppFlavors` and this https://github.com/duckduckgo/Android/blob/develop/app/build.gradle#L76

I agree, it has no place here and should be removed. I don't know if that means having a different JSON file for Apple like we do with the privacy config or what, but that needs to be resolved.

Reply:
The plan is to have separate JSON files for both, not sure if the specs are currently entirely identical though?

Reply:
I've changed the property and it's type to be the same as FeatureFlaggerInternalUserDecider.isInternalUser